tf/chunkparser.py

```diff
@@ -204,15 +192,23 @@ def convert_v4_to_tuple(self, content):
             uint8 castling them_oo (1 byte)
             uint8 side_to_move (1 byte)
             uint8 rule50_count (1 byte)
-            uint8 ply_count (1 byte)
+            uint8 dep_ply_count (1 byte) (unused)
             int8 result (1 byte)
             float32 root_q (4 bytes)
             float32 best_q (4 bytes)
             float32 root_d (4 bytes)
             float32 best_d (4 bytes)
+            float32 root_m (4 bytes)
+            float32 best_m (4 bytes)
+            float32 plies_left (4 bytes)
         """
-        (ver, probs, planes, us_ooo, us_oo, them_ooo, them_oo, stm, rule50_count, ply_count, winner, root_q, best_q, root_d, best_d) = self.v4_struct.unpack(content)
+        (ver, input_format, probs, planes, us_ooo, us_oo, them_ooo, them_oo, stm, rule50_count, dep_ply_count, winner, root_q, best_q, root_d, best_d, root_m, best_m, ply_count) = self.v5_struct.unpack(content)
```

Comment:
`ply_count` could be renamed to `plies_left` as that's what it's referred to as elsewhere.

Reply:
plies_left already used for the output variable, so I would want to rename that one if I rename ply_count to plies_left.  But I'm not sure what to rename that...

Reply:
All other outputs are also named the same as variables unpacked from struct. I don't see a reason why it couldn't be `plies_left` everywhere.

Reply:
Yeah, okay, seeems inline with the rest of the function so, done.